### PR TITLE
Floyd Warshall And Knight Walk in Python

### PR DESCRIPTION
**Pull request by atriadhiakri2000**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
YES
### Description

(Write your answer here.)

### Checklist

- [ Y ] I've read the contribution guidelines.
- [ Y ] I've checked the issue list before deciding what to submit.
- [ Y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by atriadhiakri2000** [comment]
@iamrajiv Please Check

File: Python/graphs/Floyd_Warshall_Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# Print List Function
+
+
+def printlist(table):
+    for i in table:
+        print(*i)
+FW = FloydWarshall(graph, row, col)
+print("The shortest paths are")
+format(printlist(FW))
+'''
```
Comment (iamrajiv) [inline Python/graphs/Floyd_Warshall_Algorithm.py]: @atriadhiakri2000 Remove trailing whitespace here.

File: Python/graphs/Floyd_Warshall_Algorithm.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Importing sys module to initalize the maxsize
+import sys
+''' Floyd-Warshall Algorithm '''
+
+'''Problem Statement :The problem is to find shortest distances
```
Comment (iamrajiv) [inline Python/graphs/Floyd_Warshall_Algorithm.py]: @atriadhiakri2000 Fix typos here.

File: Python/graphs/Knight_Walk.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# Importing sys module to initalize the maxsize
+import sys
+# Importing collection model to acess deque
+from collections import deque
+''' Knight-Walk '''
+'''Problem Statement :Given a chess board of order N x M and
```
Comment (iamrajiv) [inline Python/graphs/Knight_Walk.py:6]: @atriadhiakri2000 Fix typos here.

File: Python/graphs/Knight_Walk.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
+# Importing sys module to initalize the maxsize
```
Comment (iamrajiv) [inline Python/graphs/Knight_Walk.py:1]: @atriadhiakri2000 In this code lot of whitespaces is there please remove the whitespaces.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Comment by atriadhiakri2000** [comment]
@iamrajiv Please do check now

**Review by iamrajiv — APPROVED** [review]

File: Python/README.md [hunk 5]
```diff
@@ -61,6 +61,8 @@
 - [Tree (Pre Order Traversal- recursive and iterative)](ds/Tree_preOrder_traversal.py)
 - [Tree (Level Order Traversal)](ds/Tree_Level_Order_Traversal.py)
 - [Trie](ds/Trie.py)
+- [Floyd Warshall Algorithm](ds/Floyd_Warshall_Algorithm.py)
```
Comment (atarax665) [inline Python/README.md]:
```suggestion
- [Floyd Warshall Algorithm](graphs/Floyd_Warshall_Algorithm.py)
```

File: Python/README.md [hunk 6]
```diff
@@ -61,6 +61,8 @@
...
+- [Knight Walk](ds/Knight_Walk.py)
```
Comment (atarax665) [inline Python/README.md]:
```suggestion
- [Knight Walk](graphs/Knight_Walk.py)
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by atriadhiakri2000** [comment]
@HarshCasper Please check

File: Python/README.md (hunk 6 above)
Comment (atriadhiakri2000) [reply Python/README.md]: @HarshCasper Please check

**Review by atarax665 — APPROVED** [review]

**Pull request merged**